test/jasmine/tests/contour_test.js

```diff
@@ -439,21 +441,21 @@ describe('contour plotting and editing', function() {
         .then(function() {
             _assert({
                 hmIndex: 0,
-                contoursIndex: 1
+                contoursIndex: 3
```

Comment:
Since this is really about layer ordering, which is a consistently testable feature in SVG and likely to be useful elsewhere in our tests, I added a generic custom assertion for this. See what you think https://github.com/plotly/plotly.js/pull/2891/commits/e25b9ffc16a071c4f2f8c03fa5573aa6c4992a69

Reply:
> See what you think e25b9ff

:100: